diff --git a/node_modules/nightwatch/lib/core/asynctree.js b/node_modules/nightwatch/lib/core/asynctree.js
index e603b77..f8038be 100644
--- a/node_modules/nightwatch/lib/core/asynctree.js
+++ b/node_modules/nightwatch/lib/core/asynctree.js
@@ -91,8 +91,10 @@ class AsyncTree extends EventEmitter{
     if ((err.isExpect|| node.namespace === 'assert') && this.currentNode.isES6Async) {
       return true;
     }
-
-    if (this.cucumberRunner) {
+    if (err.abortOnFailure) {
+      return true;
+    }
+    if (err.waitFor) {
       return err.waitFor;
     }
 
